signature/policy_config_test.go

```diff
@@ -63,6 +63,40 @@ var policyFixtureContents = &Policy{
 	},
 }
 
+func TestDefaultPolicy(t *testing.T) {
+	// We can't test the actual systemDefaultPolicyPath, so override.
+	// TestDefaultPolicyPath below tests that we handle the overrides and defaults
+	// correctly.
+
+	// Success
+	policy, err := DefaultPolicy(&types.SystemContext{SignaturePolicyPath: "./fixtures/policy.json"})
+	require.NoError(t, err)
+	assert.Equal(t, policyFixtureContents, policy)
+
+	for _, path := range []string{
+		"/this/doesnt/exist", // Error reading file
+		"/dev/null",          // A failure case; most are tested in the individual method unit tests.
+	} {
+		policy, err := DefaultPolicy(&types.SystemContext{SignaturePolicyPath: path})
+		assert.Error(t, err)
+		assert.Equal(t, &Policy{Default: PolicyRequirements{NewPRReject()}}, policy)
```

Comment:
not sure why we want a default on error :/
it should be the caller to say: 1) got error 2) let's ask for a default 
does this sound good to you?

nitpicking though